docs/api/serializers.rst

```diff
@@ -0,0 +1,50 @@
+###########
+Serializers
+###########
```

Comment:
Agreed. Interestingly, headers used in guide are all of
```
######
Header
######
```
format. Should these be changed as well (in the separate pr maybe)?

Reply:
Sphinx documentations says
```
# with overline, for parts
* with overline, for chapters
= for sections
- for subsections
^ for subsubsections
" for paragraphs
```

It can be whatever for me, as long as it's consistent.

Reply:
#681